docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,778 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #160.                T[start_idx : stop_idx],
</pre><p>Inconsistent naming. In <code>_foreward_process</code> you use <code>start</code> and <code>stop</code> but here you use <code>start_idx</code> and <code>stop_idx</code>. Just use <code>start</code> and <code>stop</code> to make things less confusing        </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Thanks for input! I will fix it.</p>